.storybook/config.js

```diff
@@ -0,0 +1,9 @@
+import { configure } from '@storybook/react';
+
+// automatically import all files ending in *.stories.js
+const req = require.context('../stories', true, /.stories.js$/);
```

Comment:
- Can you read the stories file from `src` instead?

- The regexp should be `/\.stories\.js$/`